packages/xo-web/src/xo-app/backup/overview/tab-jobs.js

```diff
@@ -338,9 +344,22 @@ class JobsTable extends React.Component {
               {reportWhen in REPORT_WHEN_LABELS && (
                 <Li>{_.keyValue(_('reportWhen'), _(REPORT_WHEN_LABELS[reportWhen]))}</Li>
               )}
+              {backupReportTpl !== undefined && (
+                <Li>{_.keyValue(_('shorterBackupReports'), _(backupReportTpl ? 'stateEnabled' : 'stateDisabled'))}</Li>
+              )}
               {concurrency !== undefined && <Li>{_.keyValue(_('concurrency'), concurrency)}</Li>}
+              {preferNbd && <Li>{_.keyValue(_('nbdConnections'), nbdConcurrency ?? 1)}</Li>}
```

Comment:
I think there should be a message "use nbd if available" if preferNbd is enabled

Reply:
We thought that it would be a bit redundant with nbd concurrency, as it is displayed if and only if preferNbd is enabled. Do you think the clarity it brings outweights the redundancy ?

Reply:
that 's ok if you didn't forget and already handle this